tests/phpunit/tests/functions/wpGetDirectUpdateHttpsUrl.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * Tests for the wp_get_direct_update_https_url() function.
+ *
+ * @group functions.php
+ * @covers ::wp_get_direct_update_https_url
+ */
+class Tests_Functions_wpGetDirectUpdateHttpsUrl extends WP_UnitTestCase {
+
+	/**
+	 * Test that wp_get_direct_update_https_url() returns the correct URL when WP_DIRECT_UPDATE_HTTPS_URL is set.
+	 */
+	public function test_direct_update_url_with_environment_variable() {
+		putenv( 'WP_DIRECT_UPDATE_HTTPS_URL=https://example.com/update-https' );
+		$this->assertEquals( 'https://example.com/update-https', wp_get_direct_update_https_url() );
+		putenv( 'WP_DIRECT_UPDATE_HTTPS_URL=' ); // Reset the environment variable
```

Comment:
Resetting the environment variable should be done before the assertion in case of test failure.

:1234: Applies to all test methods in the PR.